terraform/environments/preview/s3_buckets.tf

```diff
@@ -212,3 +212,178 @@ resource "aws_s3_bucket" "submissions_bucket" {
     target_prefix = "digitalmarketplace-submissions-preview-preview/"
   }
 }
+
+# Cross Region replication buckets
+resource "aws_s3_bucket" "cross_region_documents_s3_bucket" {
+  provider = "aws.london"
+  bucket   = "digitalmarketplace-cross-region-documents-preview-preview"
+  acl      = "private"
+  region   = "eu-west-2"
+
+  versioning {
+    enabled = true
+  }
+
+  }
+
+  policy = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Principal": {"AWS": "arn:aws:iam::${var.aws_development_account_id}:role/replication"},
```

Comment:
The role `replication` gets its permissions from 2 places in the replication set up. The first is in the `replication_policy` attached to the role (looks good btw 👍 ) and the second is by being listed on the source bucket as the role to perform replication. We don't actually need it to have any permissions on the destination bucket other than those supplied by the role policy IIRC if it exists within the same account.